### DISCUSSION TITLE
Share stats on Postgres usage & sizing here

### DISCUSSION THREAD

**Discussion by thomasboni** [ory/kratos#1628] [Show and tell]
Hello,

I know this is a pretty hard question with a lot of "it depends" :) 

But I am trying to estimate what kind of hardware sizing I need on my postgresql database to run Kratos. I'm going to use [scaleway managed database](https://www.scaleway.com/en/pricing/#database).

So my question is the following: if you are using Kratos in production with PostgreSQL (with only Kratos using the instance), can you gave me feedback on your database usage:

- What kind of hardware sizing do you have ?
- How many users in Kratos ? 
- How many queries are sent to Kratos per <any_frequency> ? 
- \<any Kratos usage details having an impact on the db usage\>
- ...

Thanks !

**Comment by vinckr** [comment]
I moved the discussion to `Show and Tell` and changed the title to be a bit more descriptive.

**Reply by thomasboni** [reply]
Thanks, @vinckr !
